app/components/nav/index.js

```diff
@@ -145,7 +145,7 @@ class Nav extends Component {
                 display: block;
                 transform: translate(0, 69px); // Nav's height - the bottom-border
```

Comment:
I would use 70px here so the white border is still visible

Reply:
done